studio/schemas/documents/admin/jobPostings.ts

```diff
@@ -0,0 +1,21 @@
+import { defineType } from "sanity";
+
+import { jobPostingID } from "studio/schemas/objects/jobPosting";
+
+export const jobPostingsID = "jobPostings";
+
+const jobPostings = defineType({
+  name: jobPostingsID,
+  type: "document",
+  title: "Job Postings",
+  fields: [
+    {
+      name: "jobPostingsArray",
```

Comment:
I really don't like this name, but as Ane pointed out, `jobPostings` is the name of the type itself, so it might be a bit confusing if `jobPostings` consists of a single field `jobPostings` 😅 Any other name suggestions?

Reply:
I think it works, it describes what the array consists of, and the name is only for developers to use when fetching data and such, so I say go for it, unless someone has a better idea 🤷‍♀️

Reply:
agreed!